cdc/generation.hh

```diff
@@ -142,6 +142,9 @@ public:
     const std::vector<stream_id>& streams() const {
         return _streams;
     }
+    std::vector<stream_id>& streams() {
+        return _streams;
+    }
```

Comment:
Why do you want a variant that returns a non-constant vector? Aren't you afraid that the user will modify it?

Reply:
I want to modify it. Could maybe change it to a by-move-value or so, but in the stream description case, I have these by-value, and need to sort the ids. Could copy the vector, but that seems expensive.

Reply:
I see. Ok. Let's just get someone from the CDC team (@haaawk @kbr- are the usual suspects) to check that they are ok with this change, as it changes an API not specific to Alternator.

Reply:
Actually. streams_version is used just by Alternator. I would rather move the whole class out of the CDC namespace and encapsulate it in Alternator. That way there won't be any confusion.

Reply:
It still needs code in system_distributed. Moving it to alternator expands header entanglement.